.env.example

```diff
@@ -0,0 +1,5 @@
+# Application configuration values for use with docker-compose
+DOCKER_IMAGE_NAME=<docker image name, example: pulldasher:development>
+APP_PORT=<application port number>
+DEBUG_PORT=<debug port number>
+DB_PORT=<db port on host>
```

Comment:
I think these example files are a good idea, but I think it should come with defaults or a default .env file so it just works right out of the box. i.e. when running in docker-compose, don't bother opening any external ports and thus we can use the default user/password/host/database.

Same is true with the others, pick sensible defaults, but let the user override as necessary.

Reply:
Sounds good to me, I'll make that change.